Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -250,51 +250,54 @@ Set-Alias -Scope Global Rebuild-Solution Invoke-RebuildSolution
 Function Invoke-RebuildSolution {
     Param(
         [string] $buildConfiguration = "Debug",
-        [string] $target = "build",
         [string] $verbosity = "minimal",
-        [string] $maxCpuCount = "",
-        [string] $noReuse = "false",
-        [string] $platformEnvironmentVariable = "Any CPU",
-        [string] $solutionPath = (Get-RepositoryResolvedPath "Application/Ed-Fi-Ods.sln"),
-        [string] $msBuildFilePath = $env:msbuild_exe
+        [string] $solutionPath = (Get-RepositoryResolvedPath "Application/Ed-Fi-Ods.sln")
     )
     Invoke-Task -name $MyInvocation.MyCommand.Name -task {
         if ((Get-DeploymentSettings).Engine -eq 'PostgreSQL') { $buildConfiguration = 'Npgsql' }
         if (-not [string]::IsNullOrWhiteSpace($env:msbuild_buildConfiguration)) { $buildConfiguration = $env:msbuild_buildConfiguration }
 
-        $msBuildParameters = @(
-            "`"/property:Configuration=$buildConfiguration`"",
-            "`"/target:$target`"",
-            "`"/verbosity:$verbosity`"",
-            "`"/nr:$noReuse`"",
-            "`"/p:Platform=$platformEnvironmentVariable`""
-        )
-
         if (-not [string]::IsNullOrWhiteSpace($maxCpuCount)) {
-            $msBuildParameters += "`"/m:$maxCpuCount`""
+            $maxCpuCount = "/m:$maxCpuCount"
         }
         else {
-            $msBuildParameters += "`"/m`""
+            $maxCpuCount = "/m"
         }
```

Comment:
In my testing I was unable to see a difference when using max cpu with dotnet build. Lets remove this to please @tbmeador-dlp thoughts?

Reply:
I don't think we're using these variables anymore

Reply:
removed this code